SDK/AppCenterDistribute/Microsoft.AppCenter.Distribute.iOS.Bindings/ApiDefinition.cs

```diff
@@ -78,6 +78,10 @@ interface MSACDistributeDelegate
         // @optional - (BOOL)distribute:(MSACDistribute *)distribute releaseAvailableWithDetails:(MSACReleaseDetails *)details;
         [Export("distribute:releaseAvailableWithDetails:")]
         bool OnReleaseAvailable(MSACDistribute distribute, MSACReleaseDetails details);
+
+        // - (void)noReleaseAvailable;
+        [Export("noReleaseAvailable")]
+        void NoReleaseAvailable();
```

Comment:
I think it should be `OnNoReleaseAvailable(MSACDistribute distribute)`.

Reply:
Nope, it should be the same as in apple SDK.